AdminWebsite/AdminWebsite/ClientApp/src/app/services/booking-details.service.ts

```diff
@@ -87,4 +89,14 @@ export class BookingDetailsService {
         }
         return endpoints;
     }
+
+    private getInterpretee(hearingResponse: HearingDetailsResponse, participant: ParticipantResponse): string {
+        let interpreteeDisplayName = '';
+        if (participant.hearing_role_name.toLowerCase().trim() === HearingRoles.INTERPRETER) {
+            const interpreteeId = participant.linked_participants[0].linked_id;
+            const interpretee = hearingResponse.participants.find(p => p.id === interpreteeId);
+            interpreteeDisplayName = interpretee?.display_name;
```

Comment:
`interpretee?.display_name` is nice, you can use this in other places instead of
```
interpretee ? interpretee.display_name : null (or undefined, or whatever)
```